### PR TITLE
Fix errors in the users and account security section in the console

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

### Related Issues
- https://github.com/wso2/product-is/issues/16384

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx [hunk 1]
```diff
@@ -233,7 +233,7 @@ export const UsernameValidationEditPage: FunctionComponent<UsernameValidationEdi
         const usernameConf: ValidationDataInterface[] =
             validationData?.filter((data: ValidationDataInterface) => data.field === "username");
 
-        if (usernameConf === undefined || usernameConf.length < 1) {
+        if (usernameConf === undefined || usernameConf?.length < 1) {
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx:236]: We don't need the explicit undefined checking, as we are using optional chaining, do we?

```suggestion
        if (usernameConf?.length < 1) {
```

File: apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx [hunk 2]
```diff
@@ -80,7 +83,6 @@ interface WizardStepInterface {
 /**
  * Component to handle addition of a new group to the system.
  *
- * @param props props related to the create group wizard
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx:83]: Hope this is intentional :)

File: apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx [hunk 3]
```diff
@@ -320,7 +322,6 @@ export const CreateGroupWizard: FunctionComponent<CreateGroupProps> = (props: Cr
      * Handles wizard step submit.
      *
      * @param values - Forms values to be stored in state.
-     * @param {WizardStepsFormTypes} formType - Type of the form.
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx:323]: Shouldn't we keep the comment for the `formType` param?

**Review by pavinduLakshan — APPROVED** [review]

File: apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx (hunk 3 above)
Comment (DonOmalVindula) [reply apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx:323]: Fixed with aa6b04cb1d9612a46456994d7aeccf49e7b02602

File: apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx (hunk 2 above)
Comment (DonOmalVindula) [reply apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx:83]: Fixed with aa6b04cb1d9612a46456994d7aeccf49e7b02602

File: apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx (hunk 1 above)
Comment (DonOmalVindula) [reply apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx]: Fixed with aa6b04cb1d9612a46456994d7aeccf49e7b02602

**Pull request merged**